src/components/Heading/tests/Heading.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Heading from '../Heading';
+
+describe('Heading Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined', () => {
+            const headingWrapper = mount(<Heading>Online store dashboard</Heading>);
+            expect(headingWrapper.prop('children')).toBe('Online store dashboard');
+        });
+
+         it('should verify children property is not defined', () => {
+            const headingWrapper = mount(<Heading/>);
+            expect(headingWrapper.prop('children')).toBeUndefined();
+        });
+    });
+
+    describe('Test element property', () => {
+        it('should verify element property when set as h1', () => {
+            const headingWrapper = mount(<Heading element="h1">Online store dashboard</Heading>);
+            expect(headingWrapper.prop('element')).toBe('h1');
+        });
+
+         it('should verify element property when set as h2', () => {
```

Comment:
I have updated test cases and added unit test cases for how elements are rendered.So not removing that unit test cases.
Please review that.